### PR TITLE
chore: project addition by nc1z #2000

### PR DESCRIPTION
**Pull request by nc1z**
### Related Issue
Closes issue https://github.com/priyankarpal/ProjectsHut/issues/2000

### Add a new project to the list
Adding an open source React PWA Starter template, built with Create-React-App, MUI and Firebase

### Record
 - [X] I have checked the existing [issues](https://github.com/priyankarpal/ProjectsHut/issues)
 - [X] I have read the [Contributing Guidelines](https://github.com/priyankarpal/ProjectsHut/blob/main/contributing.md)
 - [X] I agree to follow this project's [Code of Conduct](https://github.com/priyankarpal/ProjectsHut/blob/main/CODE_OF_CONDUCT.md)
 - [X] I want to work on this issue

### CODE REVIEW

File: DB/projects.json [hunk 1]
```diff
@@ -5771,29 +5771,43 @@
...
+    "Social_media": {
+      "gitHub": "https://github.com/nc1z",
+      "Twitter": "https://twitter.com/nc1zdev"
+    },
+    "Projects": [
+      {
+        "link": "https://github.com/nc1z/react-pwa-starter",
+        "title": "React PWA Starter Template",
+        "description": "Opinionated React PWA Starter built with Create-React-App, MUI and Firebase",
+        "tech": ["typescript", "react", "firebase", "mui"]
```
Comment (priyankarpal) [inline DB/projects.json:5809]:
```suggestion
        "tech": ["typescript", "reactjs", "firebase", "mui"]
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]
@nc1z please add the `issue` number

**Comment by nc1z** [comment]
@priyankarpal do I create a new issue or I'm good to use the PR ticket number? Thanks

**Comment by priyankarpal** [comment]
> @priyankarpal do I create a new issue or I'm good to use the PR ticket number? Thanks

create a new issue for project addition & add the issue number to the PR description. & add `reactjs` to the tech stacks

File: DB/projects.json [hunk 2]
```diff
@@ -5771,29 +5771,43 @@
...
-    }
-  ]
-}
+  {
+    "github_username": "Aryainguz",  
+    "Social_media": {
+      "gitHub": "https://github.com/Aryainguz",
+      "LinkedIn": "https://www.linkedin.com/in/aryan-inguz-b73150254/",
+      "Instagram": "https://instagram.com/aryainguz",
+      "YouTube": "https://www.youtube.com/@thexitingway",
```
Comment (priyankarpal) [inline DB/projects.json:5780]: remove these lines, we stopped collecting `instagram` & `youtube` links

```suggestion

```

File: DB/projects.json [hunk 3]
```diff
@@ -5771,29 +5771,43 @@
...
-      "title": "Email-Newsletter-Subscription-Web-App-Using-ExpressJS",
-      "description": "Email Marketing is an important aspect of promotions of any kind, so here's a email newsletter subscription web app i made using expressjs to get subscribers for my email marketing and newsletters . Using Express JS and Mailchimp's API do use it for your personal project and share you insights with me, Aryan",
-      "tech": ["expressjs", "rest api"]
-    }
-  ]
-}
+  {
+    "github_username": "Aryainguz",  
+    "Social_media": {
+      "gitHub": "https://github.com/Aryainguz",
```
Comment (priyankarpal) [inline DB/projects.json:5777]: remove the line 

 ```suggestion
  
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by priyankarpal** [comment]
this branch has a conflicts error. Please resolve it

File: DB/projects.json [hunk 4]
```diff
@@ -5361,5 +5361,20 @@
       "tech": ["java"]
     }
   ]
-}
+},
+  {
+    "github_username": "nc1z",  
+    "Social_media": {
+      "gitHub": "https://github.com/nc1z",
```
Comment (priyankarpal) [inline DB/projects.json:5368]: remove the line also 
```suggestion

```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by nc1z** [comment]
> > There's quite a bit of formatting errors from previous project additions that have been merged to main. I've just re-added mine instead of editing others. Should be ahead of main now.
> 
> try to create separate branch in future.

Makes sense, apologies relatively new to forking & open source contributing 😄

**Review by priyankarpal — APPROVED** [review]
Thank you for contributing your projects here 👏🏻 🎉

**Comment by priyankarpal** [comment]
@nc1z  For secure collaboration, remember to check out GitHub's commit signature verification feature. It helps ensure the authenticity of commits and prevents tampering.

Learn how it works: [GitHub Docs](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification)

Happy coding! 😄

**Pull request merged**